winch/filetests/src/lib.rs

```diff
@@ -0,0 +1,167 @@
+pub mod disasm;
+
+mod test {
+    // without this, all functions are seen as dead code since they are only used in tests
+    #![allow(dead_code)]
```

Comment:
Would adding `#[cfg(test)]` above line 3 instead of this directive achieve the same result?

Reply:
Oh yes it did! Forgot to add this in, thanks